.github/workflows/inspektor-gadget.yml

```
@@ -752,6 +752,56 @@ jobs:
           echo '### Package Size Summary' >> $GITHUB_STEP_SUMMARY
           echo "$TABLE_OUTPUT" >> $GITHUB_STEP_SUMMARY
 
+  analyze-ig-memory:
+    name: IG memory analysis with pprof
+    runs-on: ubuntu-latest
+    needs: 
+      - build-ig 
+      - build-and-push-gadgets
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@3041bf56c941b39c61721a86cd11f3bb1338122a # v5.2.0
+        with:
+          go-version: '1.23.4'
+          cache: true
+        id: go
+      - name: Get ig-linux-amd64.tar.gz from artifact.
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: ig-linux-amd64-tar-gz
+          path: /home/runner/work/inspektor-gadget/
+      - name: Unpack ig-linux-amd64.tar.gz & Analyze memory
+        run: |
+          tar zxvf /home/runner/work/inspektor-gadget/ig-linux-amd64.tar.gz
+          sudo ./ig run trace_dns:latest --pprof-addr=localhost:6060 &
+          sleep 10
+          curl http://localhost:6060/debug/pprof/heap > heap.out
+          go tool pprof -top -inuse_space ./ig heap.out > inuse_space.txt
+          go tool pprof -top -alloc_space ./ig heap.out > alloc_space.txt
```

I don't see the output in the github summary but I don't see the any errors, so it is difficult to debug. Could you replace `>` by `| tee ` to help debugging?

```suggestion
          curl http://localhost:6060/debug/pprof/heap | tee heap.out
          go tool pprof -top -inuse_space ./ig heap.out | tee inuse_space.txt
          go tool pprof -top -alloc_space ./ig heap.out | tee alloc_space.txt
```